nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-process-group-configuration.js

```diff
@@ -316,6 +337,23 @@
                         $('#read-only-process-group-default-flowfile-expiration').text(processGroup.defaultFlowFileExpiration);
                         $('#read-only-process-group-default-back-pressure-object-threshold').text(processGroup.defaultBackPressureObjectThreshold);
                         $('#read-only-process-group-default-back-pressure-data-size-threshold').text(processGroup.defaultBackPressureDataSizeThreshold);
+
+                        // Determine the user-friendly name for the selected Log To Own File
+                        var selectedLogToOwnFile;
+                        if (processGroup.logToOwnFile == true) {
+                            selectedLogToOwnFile = "Yes";
+                        } else {
+                            selectedLogToOwnFile = "No";
+                        }
```

Comment:
We should use True/False, not Yes/No. This keeps us consistent with the rest of the app.